css/src/components/notification.scss

```diff
@@ -0,0 +1,85 @@
+$notification-background: $secondary-background !default;
+$notification-padding: $spacer-5 !default;
+
+$notification-font-size: $font-size-7 !default;
+$notification-color: $text !default;
+
+$notification-border-color: $control-border !default;
+$notification-border-width: $border-width !default;
+$notification-border-radius: $border-radius !default;
+
+.notification {
+	display: block;
+	position: relative;
+	padding: $notification-padding;
+	border: $notification-border-width solid $notification-border-color;
+	border-radius: $notification-border-radius;
+	outline-color: $text;
+	background-color: $notification-background;
+	color: $notification-color;
+	font-size: $notification-font-size;
+	word-wrap: break-word;
+	word-break: break-word;
+
+	@each $name, $color-set in $colors {
+		$base: nth($color-set, $color-index-base);
+		$dark: nth($color-set, $color-index-dark);
+		$background: nth($color-set, $color-index-background);
+
+		&.notification-#{$name} {
+			border-color: $base;
+			background-color: $background;
+			color: $dark;
+
+			&.is-loading::before {
+				border-color: transparent transparent $dark $dark;
+			}
+		}
+	}
+
+	&.notification-dismiss {
+		display: flex;
+		align-items: flex-start;
+		justify-content: space-between;
+	}
+
+	&.is-loading {
+		color: transparent;
+
+		> :first-child {
+			margin-inline-start: calc($notification-padding + 0.375em);
+		}
+
+		&::before {
+			@include loader;
+
+			position: absolute;
+			inset-block-start: $notification-padding;
+			inset-inline-start: $notification-padding;
+			border-color: transparent transparent $notification-color $notification-color;
+		}
+	}
+
+	.notification-title,
+	a:not(.button) {
+		color: currentColor;
+		font-weight: $weight-semibold;
+	}
+
+	.notification-title {
+		display: flex;
+		align-items: center;
+		justify-content: flex-start;
+		margin-block-end: $layout-1;
+
+		.icon {
+			flex-shrink: 0;
+			align-self: start;
+			margin-inline-end: 0.375em;
+		}
+	}
+
+	&.disappearing {
```

Comment:
Duplicative? Move out?

Reply:
Notification and banner have different animations so not really a duplicate.

Reply:
Moved out to `dismiss` file